AirportTicketBookingSystem.Test/FlightServices.Test/AddBookingTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using AirportTicketBookingSystem.Enums;
+using AirportTicketBookingSystem.FileSystem;
+using AirportTicketBookingSystem.Flights;
+using AirportTicketBookingSystem.Flights.DataModel;
+using Moq;
+using _flight = AirportTicketBookingSystem.Flights.DataModel.FlightData;
+
+namespace AirportTicketBookingSystem.Test.FlightServices.Test
+{
+    public class AddBookingTests
+    {
+        [Fact]
+        public async Task BookFlightAsync_FlightNotFound_DisplayErrorMessage()
+        {
+            // Arrange
+            var mockFileOperations = new Mock<IFileOperations>();
+            var flightManagementService = new FlightManagementService { Flights = new List<_flight>(), FileOperations = mockFileOperations.Object };
+            var consoleOutput = new StringWriter();
+            Console.SetOut(consoleOutput);
+            Console.SetIn(new StringReader("ABC123"));
+
+            // Act
+            await flightManagementService.BookFlightAsync(1);
```

Comment:
small tip: here we don't care about PassengerId so you can pass any random data using `It.IsAny<int>()`
```suggestion
            await flightManagementService.BookFlightAsync(It.IsAny<int>());
```